src/components/application_manager/src/plugin_manager/rpc_plugin_manager_impl.cc

```diff
@@ -37,7 +37,7 @@ T GetFuncFromLib(void* dl_handle, const std::string& function_name) {
       reinterpret_cast<T>(dlsym(dl_handle, function_name.c_str()));
   char* error_string = dlerror();
   if (nullptr != error_string) {
-    LOG4CXX_ERROR(logger_, "Failed to export symbols : " << error_string);
+    LOG4CXX_DEBUG(logger_, "Failed to export symbols : " << error_string);
```

Comment:
@alexkutsan why is this not considered as error?

Reply:
SDL try to load all files from specified dir. And consider that `so` is suitable is export symbols successful. So failed export is not an error, it is regular situation